src/ts/components/playbacktogglebutton.ts

```diff
@@ -28,7 +37,13 @@ export class PlaybackToggleButton extends ToggleButton<ToggleButtonConfig> {
   configure(player: PlayerAPI, uimanager: UIInstanceManager, handleClickEvent: boolean = true): void {
     super.configure(player, uimanager);
 
+    // Set enterFullscreenOnInitialPlayback if set in the uimanager config
+    if (typeof uimanager.getConfig().enterFullscreenOnInitialPlayback === 'boolean') {
+      this.config.enterFullscreenOnInitialPlayback = uimanager.getConfig().enterFullscreenOnInitialPlayback;
```

Comment:
do we need the extra config now, or can we just always use the `uimanager.getConfig().enterFullscreenOnInitialPlayback`

Reply:
With the extra config, it can still be configured per component if a customer creates their own (forked or externally configured/managed UI). So not strictly necessary but provides more flexibility.